vta/hardware/intel/ip/vta_adaptor/vta_adaptor_hw.tcl

```diff
@@ -0,0 +1,192 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# TCL File Generated by Component Editor 18.0
+# Wed Jul 03 10:20:58 CST 2019
+# DO NOT MODIFY
+
+
+# 
+# vta_adaptor "vta_adaptor" v1.0
+#  2019.07.03.10:20:58
+# 
+# 
+
+# 
+# request TCL package from ACDS 16.1
+# 
+package require -exact qsys 16.1
```

Comment:
Does QSYS 16.1 ship with Quartus Prime 18.0?

Reply:
Actually, this is generated by the Component Editor shipped with Quartus Prime 18.0, so I left it as is.